filetype/types/image.py

```diff
@@ -402,3 +402,24 @@ def match(self, buf):
                 buf[1] == 0x6F and
                 buf[2] == 0x69 and
                 buf[3] == 0x66)
+
+
+class Dds(Type):
+    """
+    Implements the DDS image type matcher.
+    """
+    MIME = 'image/dds'
+    EXTENSION = 'dds'
+
+    def __init__(self):
+        super(Dds, self).__init__(
+            mime=Dds.MIME,
+            extension=Dds.EXTENSION
+        )
+
+    def match(self, buf):
+        return (len(buf) > 4 and
+                buf[0] == 0x44 and
+                buf[1] == 0x44 and
+                buf[2] == 0x53 and
+                buf[3] == 0x20)
```

Comment:
```suggestion
        return buf.startswith(b"\x44\x44\x53\x20")
```

Reply:
Looks a bit inconsistent with other match() methods in the same file, such as in Jpeg, Gif, Png etc...

Reply:
... but fasters. Moreover, the approach is used in the repo https://github.com/h2non/filetype.py/blob/590dac5736e67882fec4344cf39343de49a3407e/filetype/types/video.py#L70
I agree with specific bytes comparison when there is "random" order.

Reply:
I see I see, thanks!